samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -10,20 +10,20 @@ Overview
 
 This sample is split into two source files:
 
-* A main file to handle initialization.
-* One additional file for handling Mesh models
+* A :file:`main.c` file to handle initialization.
+* One additional file for handling Mesh models, :file:`model_handler.c`.
 
 Provisioning
 ============
 
-Provisioning is handled by the :ref:`bt_mesh_dk_prov`.
+Provisioning is handled by the :ref:`bt_mesh_dk_prov`, with the default :file:`prj.conf` configuration.
```

Comment:
Is this addition necessary in this context? I mean, the whole document describes the sample as it behaves in its default configuration. If the users want to do this for their own app, they should instead look at the docs for bt_mesh_dk_prov directly, I think.

Reply:
Discussed f2f and removed here and in light readme.